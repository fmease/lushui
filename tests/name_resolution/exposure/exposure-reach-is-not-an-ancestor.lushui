;; TEST fail check

@(public solipsism) ;; itself
module solipsism of

@(public store.shelves) ;; descendant
module store of
    module shelves of

@(public brother) ;; sibling
module sister of

@(public locked-up.brother) ;; private relative
module brother of

module locked-up of
    module brother of
