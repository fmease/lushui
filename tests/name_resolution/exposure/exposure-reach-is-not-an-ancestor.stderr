error[E000]: exposure can only be restricted to ancestor modules
 --> ${DIRECTORY}/name_resolution/exposure/exposure-reach-is-not-an-ancestor.lushui:3:10
  |
3 | @(public solipsism) ;;; itself
  |          ^^^^^^^^^ 
  |

error[E000]: exposure can only be restricted to ancestor modules
 --> ${DIRECTORY}/name_resolution/exposure/exposure-reach-is-not-an-ancestor.lushui:6:10
  |
6 | @(public store.shelves) ;;; descendant
  |          ^^^^^^^^^^^^^ 
  |

error[E000]: exposure can only be restricted to ancestor modules
  --> ${DIRECTORY}/name_resolution/exposure/exposure-reach-is-not-an-ancestor.lushui:10:10
   |
10 | @(public brother) ;;; sibling
   |          ^^^^^^^ 
   |

error[E000]: exposure can only be restricted to ancestor modules
  --> ${DIRECTORY}/name_resolution/exposure/exposure-reach-is-not-an-ancestor.lushui:13:10
   |
13 | @(public locked-up.brother) ;;; private relative
   |          ^^^^^^^^^^^^^^^^^ 
   |

error: aborting due to 4 previous errors

